packages/fxa-shared/subscriptions/configuration/product.ts

```diff
@@ -38,6 +38,16 @@ const buildProductConfigSchema = (baseSchema: joi.ObjectSchema) =>
       'urls'
     );
 
+// This type defines the required fields of urls, set by function buildProductConfigSchema.
+// Any change to required fields in urls, should be updated here as well.
+export type ProductConfigUrlConfig = Partial<UrlConfig> & {
+  download: string;
+  privacyNotice: string;
+  termsOfService: string;
+  termsOfServiceDownload: string;
+  webIcon: string;
+};
```

Comment:
nit: instead of inlining an anon type here, we could define something with the same approach as `UrlConfig` and use the named type here.